ipwb/replay.py

```diff
@@ -992,6 +987,11 @@ def start(cdxjFilePath=INDEX_FILE, proxy=None):
         print('Sample data not pulled from IPFS.')
         print('Check that the IPFS daemon is running.')
 
+    # Check that supplied file exists
+    if not os.path.isfile(cdxjFilePath):
```

Comment:
Now you are not white-listing `ipfs:` URIs. One one hand I am not a big fan of keeping the index file in IPFS, but on the other hand this check would fail the use-case you had implemented to allow an IPFS URI as index.

Reply:
This fails multiple tests as well.

Reply:
I would instead encourage to figure out why tests are failing. There might be some assumptions made in the test file itself that is breaking here.

Reply:
I think I know what went wrong here. The following test helper function is calling the `replay.start` function on line 44, but at that point the index file `tempFilePath` is only defined and not created. It will be created on line 51. This means the `start` function of the replay does not find the index file, hence exists. A quick fix would be to create an empty file at `tempFilePath` before `replay.start`.

https://github.com/oduwsdl/ipwb/blob/c2895694c0b17df3e8179e2fe7d64d180d62b0d4/tests/test_memento.py#L37-L52